source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -76,65 +143,77 @@ Http::FilterHeadersStatus CacheFilter::encodeHeaders(Http::ResponseHeaderMap& he
 }
 
 Http::FilterDataStatus CacheFilter::encodeData(Buffer::Instance& data, bool end_stream) {
+  if (skip_encoding_) {
+    return Http::FilterDataStatus::Continue;
+  }
   if (insert_) {
-    ENVOY_STREAM_LOG(debug, "CacheFilter::encodeHeaders inserting body", *encoder_callbacks_);
+    ENVOY_STREAM_LOG(debug, "CacheFilter::encodeData inserting body", *encoder_callbacks_);
     // TODO(toddmgreer): Wait for the cache if necessary.
     insert_->insertBody(
```

Comment:
This could be a single line, unless there's some internal consistency here where this function is almost always multiple lines.

Reply:
This is actually done by the clang format script.

Reply:
Ahhh, ok. A small price to pay for not having to personally pay attention to style trivia. (And I should have remembered that the formatting is a presubmit, sorry.)

Reply:
I appreciate any comments or feedback! Sometimes actually formatting lets you have extra whitespace, so there was a chance that was a mistake indeed.